apps/storybook/stories/typography/overview.mdx

```diff
@@ -129,6 +129,11 @@ import { Label } from "@giosg/design-system-typography";
 </Label>
 `}  dark/>
 
+#### Auto Disabled Feature
+
+If you wrap your component that has disabled or data-disabled attribute, the label will be disabled automatically unless you provide color inline. For better understanding go to the - [Auto Disabled Story](/story/typography-label--auto-disabled)
```

Comment:
> the label will be disabled automatically unless you provide color inline 
i didn't get this part

Reply:
I rephrased it a bit. Give it a look when youre back